### PR TITLE
Feat form more inputs 207

### PR DESCRIPTION
**Pull request by BillyG83**
- Removed Radio from CreditCard
- Made new CreditCard picker from Radio if formatted CC data is passed
- Form can use Radio CC picker and can build formatted data to be returned
- cleaned up docs of Stepper a little

### CODE REVIEW

File: src/components/Form/Form.tsx [hunk 1]
```diff
@@ -35,38 +37,67 @@ const Form: React.FC<FormProps> = ({
...
         if (event.target.checked) {
-            optionAdded(index, opt, isRadio);
+            optionAdded(elementIndex, opt, isRadio);
         } else {
-            const filtered = data[index]?.selected?.filter(
+            const filtered = data[elementIndex]?.selected?.filter(
                 (item) => item !== opt,
             );
-            data[index].selected = filtered;
+            data[elementIndex].selected = filtered;
```
Comment (locothedev) [inline src/components/Form/Form.tsx:76]: can simply it by:
`data[elementIndex].selected = data[elementIndex]?.selected?.filter(
      (item) => item !== opt,
 );`

File: src/components/CreditCard/CreditCard.styles.ts [hunk 2]
```diff
@@ -31,23 +31,24 @@ export const DivStyledCreditCard = styled.div<TStyleProps>`
...
     ${({ isExpired, pressed }) =>
         isExpired ? expiredStyles : pressed ? pressedStyles : ''}
 `;
 
 export const DivStyledFlex = styled.div`
-    display: flex;
-    justify-content: space-between;
     align-items: center;
+    display: flex;
+    justify-content: flex-end;
```
Comment (locothedev) [inline src/components/CreditCard/CreditCard.styles.ts:41]: This style effects the creditcards footer and therefore the style does not match anymore

Should look like:
creditCardFigma

But looks like:
looksLike

**Review by locothedev — CHANGES_REQUESTED** [review]

File: src/components/Form/Form.tsx (hunk 1 above)
Comment (BillyG83) [reply src/components/Form/Form.tsx:76]: Do you think having a one liner is more simple?
I agree its less code, but I added the extra lines to make it more simple for the next dev to understand.
What do you think?

File: src/components/CreditCard/CreditCard.styles.ts (hunk 2 above)
Comment (BillyG83) [reply src/components/CreditCard/CreditCard.styles.ts:41]: ok nice, will fix now

**Pull request merged**